### PR TITLE
Add HWA docs for Rockchip VPU (RKMPP, feat. RK3588)

### PR DESCRIPTION
**Pull request by nyanmisaka**
- Add HWA docs for Rockchip VPU (RKMPP, feat. RK3588)

### CODE REVIEW

**Comment by nyanmisaka** [comment]
> Had a quick look, this is going to have merge conflicts with #875

I can wait until after that merge and make simple changes.

**Review by felix920506 — APPROVED** [review]
Will merge near 10.9 release (mid to late April)

File: docs/general/administration/hardware-acceleration/rockchip.md
Comment (felix920506) [inline docs/general/administration/hardware-acceleration/rockchip.md:9]: Add a line here that states Rockchip hw accel requires 10.9

File: docs/general/administration/hardware-selection.md
Comment (felix920506) [inline docs/general/administration/hardware-selection.md:58]: please disclaim that RK3588 is only supported in 10.9 or later

**Review by felix920506 — COMMENTED** [review]
I would like to start merging in some of the docs for 10.9 We are nearing the 10.9 release and merging this one won't hurt as long as it is disclaimed since it doesn't remove any 10.8 info.

File: docs/general/administration/hardware-selection.md
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:58]: I was told that the RK3588 doesn't have good software support on Linux. If that is the case then the wording should be clearer

File: docs/general/administration/hardware-acceleration/rockchip.md
Comment (felix920506) [reply docs/general/administration/hardware-acceleration/rockchip.md:9]: Preferably an admonition warning
```suggestion
:::warning
RKMPP Hardware Acceleration requires Jellyfin 10.9 or above
:::
```

**Review by felix920506 — APPROVED** [review]

**Pull request merged**
